hasher-matcher-actioner/hmalib/common/configs/actioner.py

```diff
@@ -48,43 +56,81 @@ class WebhookActionPerformer(ActionPerformer):
     headers: str
 
     def perform_action(self, match_message: MatchMessage) -> None:
-        self.call(data=json.dumps(match_message.to_aws()))
-
-    def call(self, data: str) -> Response:
+        parsed_url, parsed_headers = self.url, self.headers
+        for (
+            replacement_str,
+            replacement_func,
+        ) in WEBHOOK_ACTION_PERFORMER_REPLACEMENTS.items():
+            parsed_url = parsed_url.replace(
+                replacement_str, replacement_func(match_message)
+            )
+            parsed_headers = parsed_headers.replace(
+                replacement_str, replacement_func(match_message)
+            )
+        self.call(parsed_url, parsed_headers, data=json.dumps(match_message.to_aws()))
+
+    def call(self, url: str, headers: str, data: str) -> Response:
         raise NotImplementedError()
 
 
 @dataclass
 class WebhookPostActionPerformer(WebhookActionPerformer):
     """Hit an arbitrary endpoint with a POST"""
 
-    def call(self, data: str) -> Response:
+    def call(self, url: str, headers: str, data: str) -> Response:
         logger.info(f"Performing a POST request to URL. {self.url}")
-        return post(self.url, data, headers=json.loads(self.headers))
+        return post(url, data, headers=json.loads(headers))
 
 
 @dataclass
 class WebhookGetActionPerformer(WebhookActionPerformer):
     """Hit an arbitrary endpoint with a GET"""
 
-    def call(self, data: str) -> Response:
+    def call(self, url: str, headers: str, data: str) -> Response:
         logger.info(f"Performing a GET request to URL. {self.url}")
-        return get(self.url, headers=json.loads(self.headers))
+        return get(url, data, headers=json.loads(headers))
```

Comment:
Did you test get or delete? I don't think either requests.get or .delete expects 'data' as the requests don't have bodies at least usually.